extensions/adobe/experience/customerJourneyManagement/email-channel-context.schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/experience/customerJourneyManagement/email-channel-context",
```

Comment:
All the schema extensions are created under the hierarchy "adobe/experience/customerJourneyManagement" but the "id" field has "xdm" in path. This was require, as per the test scripts the id field has to follow the regex:
`should match pattern "(https://ns\.adobe\.com/xdm/[a-z0-9-/]*)|(http://schema\.org/.*)|(http://ns.adobe.com/adobecloud/core/1.0.*)|([http|https]://ns\.adobe\.com/adobecloudplatform/.*)|(https://tools\.ietf\.org/html/draft-kelly-json-hal-08/.*)|(http://www.iptc\.org/.*)|(https://id3\.org/id3v2.4/.*)`